src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.html

```diff
@@ -40,14 +40,14 @@
           </app-configuration-group>
           <div class="columns">
             <div class="column">
-              <div class="field is-horizontal" [ngClass]="{'enable-div': isServiceAvailable}">
-                <div class="field-label has-text-left" [ngClass]="{'service-label':!isServiceAvailable === true}">
+              <div class="field is-horizontal" [ngClass]="{'enable-div': serviceInfo.added}">
+                <div class="field-label has-text-left" [ngClass]="{'service-label':!serviceInfo.added === true}">
                   <label class="label">Enabled</label>
                 </div>
                 <div class="field-body">
                   <div class="field">
-                    <input class="checkbox" name="enabled" type="checkbox" [checked]="isServiceEnabled" [value]="isServiceEnabled"
-                      [(ngModel)]="isServiceEnabled" [disabled]="!rolesService.hasEditPermissions()">
+                    <input class="checkbox" name="enabled" type="checkbox" [checked]="enabled" [value]="enabled"
+                      [(ngModel)]="enabled" [disabled]="!rolesService.hasEditPermissions()">
```

Comment:
When we are using two way binding using [(ngModel)], we don't need [value] attribute on the field.